main.js

```diff
@@ -73,6 +73,13 @@ function reminderTour(win){
 
 function firstInstallTour(win) {
   if (win.activeInstall || win.reminderTour) {
+    if (get('extensions.tabcentertest1@mozilla.com.tourComplete')) {
+      utils.sendPing('tour_completed_reminder_began', win);
```

Comment:
Also, I think we can set a `details` variable to `'install'`, `'reminder'`, or `'completed_reminder'` way up here, and just call `utils.sendPing(<whatever>, win, details);` in the rest of the code below.  🙂